### ISSUE TITLE
list_transcripts working locally, but not from server

### ISSUE THREAD

**Issue by mgoldenbe** [jdepoix/youtube-transcript-api#262]
The following works locally just fine:

```py
from youtube_transcript_api import YouTubeTranscriptApi
YouTubeTranscriptApi.list_transcripts('JdcmfiMS-84')
```

When I run the same code from the console in PythonAnywhere, I get:

```
Could not retrieve a transcript for the video https://www.youtube.com/watch?v=JdcmfiMS-84! This is most likely caused by:

Subtitles are disabled for this video
```

What could be causing this?

**Comment by jdepoix** [comment]
Hi @mgoldenbe, when I try to open the video I get the error message "Video unavailable. The uploader has not made this video available in your country" (I am currently in France). It seems likely that the server you are running your code on is not located in the same country as you are and therefore is blocked due to the same reason.

The error message is a bit misleading here, as I hadn't encountered this case yet. I will try to add a more explicit error message for this in the future. I will close this issue now as there's not much we can do here to solve this specifically.

**Comment by mgoldenbe** [comment]
Yes, it would be great if I could determine this somehow. Interestingly, I
am able to get the listing for this video using YouTube List API from the
same server...
